src/EditorFeatures/Test/Utilities/PatternMatcherTests.cs

```diff
@@ -528,9 +512,10 @@ private static IEnumerable<PatternMatch> TryMatchMultiWordPattern(string candida
         {
             MarkupTestFile.GetSpans(candidate, out candidate, out ImmutableArray<TextSpan> expectedSpans);
 
-            var matches = new PatternMatcher(pattern).GetMatches(candidate, includeMatchSpans: true);
+            var matches = ArrayBuilder<PatternMatch>.GetInstance();
```

Comment:
In general, it would be a "doesn't matter". However, we shoudl be calling 'free' here just so we get coverage that free isn't working improperly.